docs/whats-new/preview-4.md

```diff
@@ -0,0 +1,699 @@
+
+- `RunAsXX`: Only affects the model when running locally.
+- `PublishAsXX`: Only affects the model when publishing the manifest.
+- `AsXX`: Affects the model both when running locally and when publishing the manifest.
+
+#### Examples
+
+The following logic will use a redis container locally and prompt for
+the connection string when deploying.
+
+```csharp
+var builder = DistributedApplication.CreateBuilder(args);
+
+var redis = builder.AddRedis("redis")
+                   .PublishAsConnectionString();
+
+builder.AddProject<Projects.WebApplication1>("api")
+       .WithReference(redis);
+
+builder.Build().Run();
+```
+
+## General application model improvements
+
+There are several general improvements to the application model that make it easier to model common scenarios.
+
+### Changing container properties
+
+We added some new methods to tweak container images, tags and volumes.
+
+Here's an example of using the "latest" image tag for the [redis image from DockerHub](https://hub.docker.com/_/redis/).
+
+```csharp
+var builder = DistributedApplication.CreateBuilder(args);
+
+var redis = builder.AddRedis("redis")
+                   .WithImageTag("latest"); // Prefer the latest
+
+builder.AddProject<Projects.WebApplication1>("api")
+       .WithReference(redis);
+
+builder.Build().Run();
+```
+
+### Split bind mounts and volumes into separate methods
+
+Splitting bind mounts and volumes into separate methods. Bind mounts are used to mount a file or directory from the host into the container. Volumes are used to mount a volume from the host into the container. Splitting these into separate methods makes it easier to understand how the container is being used.
+
+```csharp
+var builder = DistributedApplication.CreateBuilder(args);
+
+// MySql database and table names are case-sensitive on non-Windows.
+var catalogDbName = "catalog";
+
+var catalogDb = builder.AddMySql("mysql")
+    .WithEnvironment("MYSQL_DATABASE", catalogDbName)
+    .WithBindMount("../MySql.ApiService/data", "/docker-entrypoint-initdb.d")
+    .AddDatabase(catalogDbName);
+
+builder.AddProject<Projects.MySql_ApiService>("api")
+       .WithReference(catalogDb);
+
+builder.Build().Run();
+```
+
+### Addition of more database admin tools
+
+.NET Aspire Preview 3 introduced the ability to manage postgres databases using pgAdmin and redis using redis commander. Preview 4 introduces the ability to manage MySql databases using phpMyAdmin, and MongoDB databases using mongo-express.
+
+```csharp
+var builder = DistributedApplication.CreateBuilder(args);
+
+var mongo = builder.AddMongoDB("mongo")
+                   .WithMongoExpress()
+                   .AddDatabase("db");
+
+var mySql = builder.AddMySql("mysql")
+                   .WithPhpMyAdmin()
+                   .AddDatabase("catalog");
+
+builder.AddProject<Projects.WebApplication1>("api")
+       .WithReference(mongo)
+       .WithReference(mySql);
+
+builder.Build().Run();
+```
+
+## Ability to ignore launch profiles
```

Comment:
I think we should consider removing this section. We made some changes in preview 5 that makes this API go away. So it seems strange to call it out here only to get rid of it.

Reply:
Yes, if that's the case - I agree. I'll remove it.